docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+
+services:
+    # Service for Python
+  backend:
+    container_name: flare-backend
+    build:
+      context: .    
+      dockerfile: Dockerfile.backend  
+    restart: always
+    command: ["tail", "-f", "/dev/null"]
+    volumes:
+      - ./data:/app/data:z  # Shared volume for CSV files
+    networks:
+      - broadcast-network
+
+  # Service for Vue app with Nginx
+  frontend:
+    container_name: flare-frontend
+    build:
+      context: .   
+      dockerfile: Dockerfile.frontend 
+    restart: always
+    ports:
+      - "${FRONTEND_HOST_PORT}:80"  # Host Port:Container Port
```

Comment:
I don't see a .env.dist or a README instruction to create a `.env` file so from a standpoint of someone new to a project, I don't know where we get the value for this environment variables